### ISSUE TITLE
Awacs and Wingman Addons

### ISSUE THREAD

**Issue by TBarina** [chump29/DCS_Mod#3]
Hi Chum29. Great mod indeed. It's a pity it has gone.

I'm guessing you had to remove it due to ED changes.
Out of curiosity, why is it no longer possible to manage at least the spam of the wingman's contact reports?

All the best

**Comment by TBarina** [comment]
I've modified Scripts\Speech\common.lua and NATO.lua for DCS 2.9.7.59263 in my DCS installation folder  to reintroduce WIngman Addon's message spam functionalities. it seems to work flawlessly. 
I've attached the files here. Hope you don't mind.
It is exactly what I was searching for.
Don't understand why ED doesn't take in account your  great job. You are a genius... :)
Cheers
[Chump29WingmanAddonMod.zip](https://github.com/user-attachments/files/17011611/Chump29WingmanAddonMod.zip)

**Comment by Amarok73** [comment]
@TBarina Because of the errors as in screenshot, is it safe to delete from common.lua everything from line 3319 down?
![obraz](https://github.com/user-attachments/assets/283c3580-dd8f-4d41-9fca-dd8421043f0c)

**Comment by TBarina** [comment]
Yes, you are right. I used the common.lua in my Eagle Dynamics\DCS World OpenBeta\Scripts\Speech directory and didn't notice that Vaicom added something there.
You can remove if you don't use Vaicom.

**Comment by TBarina** [comment]
We hope that ED takes it upon themselves to include this valuable improvement made by Chump29.
I'm a little disappointed to see that they keep releasing new planes and terrain and not fixing these things and the numerous others that completely ruin the immersion of the simulation
